pkg/artifactory/resource/repository/federated/resource_artifactory_federated_repository_test.go

```diff
@@ -149,7 +149,7 @@ func TestAccFederatedRepoWithProjectAttributesGH318(t *testing.T) {
 	projectEnv := test.RandSelect("DEV", "PROD").(string)
 	repoName := fmt.Sprintf("%s-generic-federated", projectKey)
 
-	_, fqrn, name := acctest.MkNames(repoName, "artifactory_federated_generic_repository")
+	_, fqrn, name := acctest.MkNames(repoName, "artifactory_federated_terraform_provider_repository")
```

Comment:
Why are we changing the name of the resource type? If you need to test TF repo type, then we should have additional, separate test.

Reply:
I'll revert it, all TF packages will be tested in `TestAccFederatedRepoAllTypes`